superbench/cli/_commands.py

```diff
@@ -59,4 +61,18 @@ def load_arguments(self, command):
                 nargs='+',
                 help='Extra arguments to override config_file.'
             )
+        with ArgumentsContext(self, 'result') as ac:
+            ac.argument(
+                'raw_data_file', options_list=('--raw-data-file', '-d'), type=str, help='Path to raw data file.'
```

Comment:
`raw-data-file` or `data-file` directly?

Reply:
python variable is using `_` instead of `-`

Reply:
Don't get the point. I am talking about options_list

Reply:
change --raw-data-file to -data-file